app/Http/Controllers/UpdateAirlineController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CreateAirlineRequest;
+use App\Models\Airline;
+
+class UpdateAirlineController extends Controller
+{
+    public function __invoke(CreateAirlineRequest $request, Airline $airline)
+    {
+        return $airline->update($request->validated());
+    }
```

Comment:
```suggestion
    public function __invoke(UpdateAirlineRequest $request, Airline $airline): JsonResponse
    {
        $airline->update($request->validated())
    
        return response()->json($airline);
    }
```
The `update` method returns a boolean, that's why I'm returning the `$airline` instead of the result from the method in the suggestion.
As for the FormRequest, you should definitely have a different one because there's a difference between them (See comments in the `CreateAirlineRequest` to learn more about the difference).